### PR TITLE
Nc news 4 get api/articles/:article_id

### PR DESCRIPTION
**Pull request by The-Nightman**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -33,16 +33,48 @@ describe("get /api", () => {
...
+            });
+    });
+    test("returns a status 404 error message when an inavlid article id is searched", () => {
+        return request(app)
+            .get("/api/articles/1528")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.message).toBe('Article does not exist!')
+            });
+    });
```
Comment (MKD01) [inline __tests__/app.test.js:73]: This is a good test that ensures an id that is valid but does not exist is given a 404 status with an appropriate message, but what if a completely invalid id is given? What would the expect behaviour for this be?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -33,16 +33,48 @@ describe("get /api", () => {
...
+            "article_img_url"]
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+                articleProps.forEach(prop => {
+                    expect(body.article).hasOwnProperty(prop);
+                })
+            });
+    });
```
Comment (MKD01) [inline __tests__/app.test.js:65]: This endpoint is for a specific article so we can and should be more specific with the expect statements. We know that the id should be 1, we also know that each of the other properties should exist on this object as well and we can also assert what these are. I would suggest taking a look at expect.objectContaining or .toMatchObject as well.

**Review by MKD01 — CHANGES_REQUESTED** [review]
Good work so far but there are a few things I would like to see changed/added for this ticket. Once you have made those changes please re-submit this pr

**Review by robert112233 — APPROVED** [review]
Great work, you've taken on board the feedback you've implemented all the necessary changes, merge away 🏅

**Pull request merged**